specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Pipeline.json

```diff
@@ -2790,6 +2798,14 @@
         "additionalColumns": {
           "type": "object",
           "description": "Specifies the additional columns to be added to source data. Type: array of objects(AdditionalColumns) (or Expression with resultType array of objects)."
+        },
+        "numberPrecision": {
+          "type": "object",
+          "description": "The decimal precision used to represent Oracle NUMBER type without precision and scale. The range is 1 to 256 and default value is 256 if not specified. Type: integer. Only used for Version 2.0."
+        },
+        "numberScale": {
+          "type": "object",
```

Comment:
>object [](http://example.com/codeflow?start=19&length=6)

integer

Reply:
In ADF, the value could be literal or expressions that are evaluated at our runtime, so here we use object instead of integer. You can also see that this whole file is using object.
Reference: https://learn.microsoft.com/en-us/azure/data-factory/control-flow-expression-language-functions